src/buffer.h

```diff
@@ -150,6 +150,11 @@ class Buffer {
   /// initial count. This requires the format to have been set.
   void ResizeTo(uint32_t element_count);
 
+  /// Resizes the buffer to hold |element_count| elements. This requires the
+  /// format to have been set. This is separate from ResizeTo(). The given
+  /// argument here is |size_in_bytes| bytes vs |element_count| elements
+  void ResizeToBytes(uint32_t size_in_bytes);
```

Comment:
Should this be called SetSizeInBytes and be put up above with GetSizeInBytes? Not really sure, I named `ResizeTo` poorly .....